### PR TITLE
Updating numpy requirements

### PR DESCRIPTION
**Pull request by juanvergaramunoz**
(1) presales_evaluation.ipynb was breaking with older np version; (2) additionally updating pull request template

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-numpy==1.15.2
+numpy==1.16.4
```
Comment (journeyman-msft) [inline requirements.txt:1]: AH i see you used requirements.txt locally. environment.yml was on par. These should be consistent, yes.

**Review by journeyman-msft — APPROVED** [review]

File: requirements.txt (hunk 1 above)
Comment (juanvergaramunoz) [reply requirements.txt:1]: Correct. I checked environment.yml, which seemed to be okay. Didn't checked other dependencies, don't accept just yet. I will check the other libraries (seaborn seems to be missing, for instance)

Comment (juanvergaramunoz) [reply requirements.txt:1]: Finalized comparing requirements.txt and environment.yml   ---   Re-requested review

File: requirements.txt [hunk 2]
```diff
@@ -1,15 +1,24 @@
...
 microsoft-bonsai-api==0.1.2
-bonsai-cli==1.0.1
-vpython==7.6.0
\ No newline at end of file
+bonsai-cli==1.0.4
+vpython==7.6.0
+cufflinks==0.16
+joblib==0.13.2
+pyyaml==5.3.1
+h5py==2.10.0
```
Comment (journeyman-msft) [inline requirements.txt:23]: good catch on the h5py, i did not include this as an oversight from my previous bugfix.

**Review by journeyman-msft — APPROVED** [review]

**Pull request merged**